app/src/test/java/org/pesmypetcare/mypetcare/services/StubCommunityService.java

```diff
@@ -55,7 +55,7 @@ private static void addForums() {
         forum.addTag("important");
         new Forum("Washing", "John Doe", DateTime.Builder.buildFullString("2020-04-22T10:00:00"),
             StubCommunityService.groups.get(DINOSAURS));
-        new Forum("Sickling", "John Doe", DateTime.Builder.buildFullString("2020-04-22T10:10:00"),
+        Forum forum2 = new Forum("Sickling", "John Doe", DateTime.Builder.buildFullString("2020-04-22T10:10:00"),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Distance between variable 'forum2' declaration and its first usage is 5, but allowed 4.  Consider making that variable final if you still need to store its value in advance (before method calls that might have side effects on the original value).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5556584)